CHANGELOG.md

```
@@ -1,15 +1,24 @@
 # Prowide Core - CHANGELOG
 
-#### 9.4.19 - SNAPSHOT
+#### 9.5.3 - SNAPSHOT
```

Markdownlint

Expected: h2; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)